src/lib/middleware/same_origin.coffee

```diff
@@ -3,5 +3,7 @@
 #
 
 module.exports = (req, res, next) ->
-  res.set('X-Frame-Options', 'SAMEORIGIN')
+  if !res.headersSent
+    res.set('X-Frame-Options', 'SAMEORIGIN')
```

Comment:
There shouldn't be any risk of this happening after the response has begun, right? (I'd hope it's safe based on middleware order.)